src/test/java/de/flapdoodle/embed/process/store/LocalArtifactStoreTest.java

```diff
@@ -0,0 +1,113 @@
+/**
+ * Copyright (C) 2011
```

Comment:
Should I update the copyright to 2018 since it's a new file?

Reply:
hmm.. not sure.. i will leave this at is is..